include/HPDDM_inexact_coarse_operator.hpp

```diff
@@ -737,8 +737,7 @@ class InexactCoarseOperator : public OptionsPrefix<K>, public Solver
             }
             delete [] _rq;
 #else
-            if(_idx)
-                PetscFree(_idx);
+            PetscFree(_idx);
```

Comment:
Where is it documented that this function tolerates also the passing of null pointers?

Reply:
Here: https://petsc.org/main/docs/manualpages/Sys/PetscFree/#notes.